mousestyles/data/__init__.py

```diff
@@ -1,15 +1,46 @@
 import os as _os
-
 import numpy as np
-
+import pandas as pd
 from mousestyles import data_dir
 
-def all_feature_data():
+def load_all_features():
+    """
+    Returns a (21131, 13) size pandas.DataFrame object corresponding to
+    9 features over each mouse's 2-hour time bin. The first four columns
+    index each mouses's 2-hour bin:
+
+    Column 0: the strain of the mouse (0-15)
+    Column 1: the mouse number (number depends on strain)
+    Column 2: the day number (5-16)
+    Column 3: 
 
-    feat_arr = ['ASProbability', 'ASNumbers', 'ASDurations', 'Food', 'Water', 'Distance', 'ASFoodIntensity', 'ASWaterIntensity', 'MoveASIntensity']
-    feat_arr_units = ['Active state probability', 'Number AS onsets', 'Total time [sec]', 'Food consumed [g]', 'Water consumed [mg]', 'Distance travelled [m]', 'ASFoodIntensity', 'ASWaterIntensity', 'MoveASIntensity [cm/ASsec]']
+    The remaining 9 columns are the computed features.
+
+    Returns
```

Comment:
We should settle on a docstring convention tomorrow and make sure that it's clearly outlined in the contributor doc, as you suggested in #17.